### DISCUSSION TITLE
Property list: return `path` if `playlist-path`  is empty

### DISCUSSION THREAD

**Discussion by lamyergeier** [mpv-player/mpv#14585] [Q&A] [answered]
From MPV [Property list](https://mpv.io/manual/master/#property-list), there are 2 variables: `path` and `playlist-path` .

In `input.conf`,  I did the following: (it works for a playlist file)

```
# Open the media in vlc
ctrl+v run "/bin/bash" "-c" "/usr/bin/vlc \"${playlist-path}\""; show-text "Open in VLC" 2000
```

But if the current media being played is not a part of playlist then above won't work. Is it possible if the `playlist-path` is empty it can be automatically be substituted by `path`?

**Comment by zhongfly** [comment, marked as answer]
https://mpv.io/manual/master/#property-expansion

`${playlist-path:${path}}`
